README.md

```diff
@@ -279,3 +279,11 @@ $ cosa init https://github.com/coreos/fedora-coreos-config.git
 $ cosa fetch && cosa build
 ```
 See this [Stack Overflow question](https://stackoverflow.com/questions/26028971/docker-container-ssl-certificates) for additional discussion.
+
+#### Overriding RPMs locally
```

Comment:
Can we update this to say 'Overriding RPMS: Using new, different, or locally built RPMS'

Reply:
Updated!